.travis.yml

```diff
@@ -1,9 +1,33 @@
 language: python
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
+  - "2.7.12"
+  - "3.5"
+
+install:
+  - pip install -r requirements.txt
+  - pip install flake8
+
+before_script:
+  # --exit-zero means that flake8 will only warn and not stop the build
```

Comment:
Could we drop most of this into a config file for flake8?  Possibly in the "files" subdirectory?

We could then suggest git commit hooks to ensure we don't get any worse according to our chosen standards...